docs/userGuide/plugins/web3Form.md

```diff
@@ -0,0 +1,108 @@
+### Plugin: Web3Forms 
+
+This plugin allows you to create forms whose response will be sent directly to your email, using the [Web3Form](https://web3forms.com/) API. 
+<box type="warning" seamless>
+Emails from Web3Forms may end up in spam/junk mail.
+</box>
+
+<box type="warning" seamless>
+The free plan from Web3Form only allows 250 submissions per month.
```

Comment:
```suggestion
The free plan from Web3Forms only allows 250 submissions per month.
```
Nit, can we standardise to use "Web3Forms" throughout? Same for the rest of the docs

Reply:
```suggestion
The free plan from Web3Form allows up to 250 submissions per month.
```
Another nit, to sound more optimistic 😄